[Autocomplete] Allow to edit route (#625)
symfony/ux: src/Autocomplete/doc/index.rst

    these won't be used during the Ajax call to fetch results. Instead, include
    all options inside the custom class (``FoodAutocompleteField``).

.. note::

>     .. versionadded:: Unreleased

Reviewer: A few small changes here:

1) Use 2.7 for the version. That will at least probably be correct - so better than `Unreleased` :) 
2) Make the whole thing look like this:

```
.. versionadded:: 2.7

    The ability to specify the route was added in Twig Components 2.7.

The default route...
```

So, a small indented message for the `versionadded`, then go onto the description.

>     The default route for Autocomplete components is ``/autocomplete/{alias}/``

Reviewer: ```suggestion
    The default route for the Ajax calls used by the Autocomplete component is ``/autocomplete/{alias}/``
```

>     Sometimes it may be useful to customize this URL - e.g. so that the component lives

Reviewer: ```suggestion
    Sometimes it may be useful to customize this URL - e.g. so that the URL lives
```

    under a specific firewall.

>     To use another route, you need to declare it :

Reviewer: ```suggestion
    To use another route, you need to declare it:
```

>     .. code-block:: yaml

Reviewer: Add an empty line between `code-block` and the actual code - that's just something we always do

>         # config/routes/attributes.yaml

Reviewer: ```suggestion
        # config/routes.yaml
```

        ux_entity_autocomplete_admin:
          controller: ux.autocomplete.entity_autocomplete_controller
          path: '/admin/autocomplete/{alias}'

>     then specify this new route to your component :

Reviewer: ```suggestion
    Then specify the route in the attribute:
```

>     .. code-block:: diff

Reviewer: New line below this

        // src/Form/FoodAutocompleteField.php
        use Symfony\UX\Autocomplete\Form\AsEntityAutocompleteField;
        - #[AsEntityAutocompleteField]

>         + #[AsEntityAutocompleteField(route: 'ux_entity_autocomplete_admin')]

Reviewer: I DO like `diff` code blocks, but in this case, it's so simple, I think we can make this a PHP code block and JUST show this line and the `class` line below (no `use` statement really needed here, as the user will already have the attribute).
